recipes/calculix/Makefile_MT_windows

```diff
@@ -0,0 +1,34 @@
+
+CFLAGS = -Wall -O3 -fopenmp -I $(LIBRARY_PREFIX)/include/spooles/ -DARCH="Linux" -DSPOOLES -DARPACK -DMATRIXSTORAGE -DUSE_MT=1
+FFLAGS = -Wall -O3 -fopenmp
+
+CC=gcc
+FC=gfortran
+
+.c.o :
+	$(CC) $(CFLAGS) -c $<
+.f.o :
+	$(FC) $(FFLAGS) -c $<
+
+include Makefile.inc
+
+SCCXMAIN = ccx_2.14.c
+
+OCCXF = $(SCCXF:.f=.o)
+OCCXC = $(SCCXC:.c=.o)
+OCCXMAIN = $(SCCXMAIN:.c=.o)
+
+DIR=..\..\..\SPOOLES.2.2
+
+LIBS = \
+       $(LIBRARY_PREFIX)/lib/spoolesMT.a \
+       $(LIBRARY_PREFIX)/lib/spooles.a \
+       $(LIBRARY_PREFIX)/mingw-w64/lib/libarpack.a \
```

Comment:
Why static here?

Reply:
There are not many references for this library, but it was suggested here: ![https://mechanicalhacks.wordpress.com/2011/03/29/building-calculix-to-run-native-on-64-bit-windows/](https://lh6.googleusercontent.com/BxEHCaLdq4zq-2Ug6JyQ5TDSGxnRJbOHft69h7SKq_vU2o2jCjqWza9z0k2Mrv_LmKDJ1jJCGhIXCLBepJlYiymywG_KfAOAmua7jqVS08Tbl6_1sxU)
https://mechanicalhacks.wordpress.com/2011/03/29/building-calculix-to-run-native-on-64-bit-windows/

Reply:
@isuruf I guess arpack is only available as a static library for windows. And it's better to have consistent run-deps for the other os.

Reply:
Please send a PR to arpack to build it as a shared library.